src/shared/options/readOptionDefaults/parsePackageAuthor.ts

```diff
@@ -1,12 +1,11 @@
+import parse from "parse-author";
+
 import { PartialPackageData } from "../../types.js";
 
 export function parsePackageAuthor(packageData: PartialPackageData) {
 	const [packageAuthor, packageEmail] =
 		typeof packageData.author === "string"
-			? [
-					packageData.author.split("<")[0].trim(),
-					packageData.author.split(/<|>/)[1]?.trim(),
-			  ]
+			? [parse(packageData.author).name, parse(packageData.author).email]
```

Comment:
Good direction! I'd rather this only `parse` once though. Might have to start putting an `if` statement or two in 😄

Reply:
Ok thanks for the feedback - I'll look into this!

Reply:
Hey Josh - I reconfigured the code to include an if statement and a variable to hold the parsed author information. Let me know if you like how it looks.